src/opera/parser/tosca/csar.py

```diff
@@ -79,16 +326,16 @@ def package_csar(self, output, service_template=None, csar_format="zip"):
                 # with python version lower than 3.8)
                 tempdir = mktemp(prefix="opera-package-")
                 # create tempdir and copy in all the needed CSAR files
-                shutil.copytree(self._csar_name, tempdir)
+                shutil.copytree(self.csar_dir, tempdir)
 
                 # create TOSCA-Metadata/TOSCA.meta file using the specified
                 # TOSCA service template or directory root YAML file
                 content = """
-                          TOSCA-Meta-File-Version: 1.1
-                          CSAR-Version: 1.1
-                          Created-By: xOpera TOSCA orchestrator
-                          Entry-Definitions: {}
-                          """.format(service_template)
+TOSCA-Meta-File-Version: 1.1
+CSAR-Version: 1.1
+Created-By: xOpera TOSCA orchestrator
+Entry-Definitions: {}
```

Comment:
Wow, are these lines left-drunk? Please reformat.

Reply:
This is intentional. Previously, every line in the meta file had spaces in front, which worked because YAML ignores consistent leading spaces. However, this isn't a proper TOSCA meta file - per the standard, it should have been parsed as (converting to JSON for clarity:

```json
{
    "                    TOSCA-Meta-File-Version": "1.1Csar-Version: 1.1CreatedBy: xOpera TOSCA orchest...."
}
```

Reply:
Aha I see. Yes `TOSCA.meta` is not a YAML. But the code looked really strange as this was moved to the left.

Reply:
I changed this to concatenated single-quoted strings. It now no longer sticks to the left.